backend/src/v5/services/sessions.js

```diff
@@ -31,6 +32,7 @@ const initialiseSession = async () => {
 
 	// istanbul ignore next
 	const middleware = expressSession({
+		genid: () => generateUUIDString(),
```

Comment:
Can we generate a uuid (not string), then represent it in base64? so it's shorter